docs/functions.rst

```diff
@@ -1573,6 +1573,41 @@ Example:
           name: "test-snapshot-content-content-dfc8fa67-8b11-4fdf-bf94-928589c2eed8"
 
 
+DeleteDataUsingKopiaServer
+----------
+
+This function deletes the snapshot data backed up by the BackupDataUsingKopiaServer function.
+It creates a new Pod that runs delete snapshot job created using Kopia Repository Server
+
+.. csv-table::
+   :header: "Argument", "Required", "Type", "Description"
+   :align: left
+   :widths: 5,5,5,15
+
+   `namespace`, Yes, `string`, namespace in which to execute
+   `image`, Yes, `string`, image to be used for running delete
```

Comment:
Let's improve these acc to https://github.com/kanisterio/kanister/pull/1857#discussion_r1121344308